screens/login.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
```

Comment:
It looks like you checked out this branch from your `login` branch - please make sure that you checkout only from the `master` branch. If you don't, you include code that isn't technically approved to be merged into another.

Remove at files that are connected to login - that includes other js or assets files.